docs/source/administrator/services.md

```diff
@@ -19,7 +19,7 @@ In the following snippet, I'm using a custom image that copies over the applicat
 ```Dockerfile
 # Dockerfile
 # 2.0.0 is latest stable release at the time of this writing
-FROM jupyterhub/k8s-hub:2.0.0
+FROM quay.io/jupyterhub/k8s-hub:2.0.0
```

Comment:
Might as well update this and the above comment to `3.1.0`?

Reply:
Fixed, and left a link to the tags.